controllers/machine_controller_phases_test.go

```diff
@@ -239,18 +242,21 @@ var _ = Describe("Reconcile Machine Phases", func() {
 		err = unstructured.SetNestedField(infraConfig.Object, "test://id-1", "spec", "providerID")
 		Expect(err).NotTo(HaveOccurred())
 
+		// Set NodeRef.
```

Comment:
I can't comment on the line in question, but the description for this `It` case needs to be updated since we're expecting `Running` now

Reply:
sorry missed that one, fixed now